cirq-core/cirq/ops/three_qubit_gates.py

```diff
@@ -49,9 +49,22 @@
 
 
 class CCZPowGate(gate_features.InterchangeableQubitsGate, eigen_gate.EigenGate):
-    """A doubly-controlled-Z that can be raised to a power.
-
-    The matrix of `CCZ**t` is `diag(1, 1, 1, 1, 1, 1, 1, exp(i pi t))`.
+    r"""A doubly-controlled-Z that can be raised to a power.
+
+    The unitary matrix of `CCZ**t` is
+
+    $$
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+        0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
+        0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 \\
+        0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 \\
+        0 & 0 & 0 & 0 & 0 & 0 & 0 & e^{i \pi t}
```

Comment:
(This is a nice example showing the advantage of implicit zeros I advocated for in another comment. It takes some eye-strain to recognize that this is diagonal.)

Reply:
done here for 3 qubit gates, but left 2 qubit gates in #5359